.github/workflows/reusable-build-test-release.yml

```diff
@@ -77,6 +92,24 @@ concurrency:
   group: ${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 jobs:
+  validate-custom-version:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.inputs.custom-version != '' }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Validate custom version
+        run: |
+          if [[ ! ${{ github.event.inputs.custom-version }} =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
Just please decide wether "v" is required before input version string. Document that to avoid confusion.

Reply:
Added a note about custom version format in the [input description](https://github.com/splunk/addonfactory-workflow-addon-release/blob/efb57ff3a6148aad25cda912c3d0511ee783eaef/.github/workflows/reusable-build-test-release.yml#L19)

We don't need "v" in this line as we're just validating the provided string, which doesn't contain "v". In the existing version validation we need "v" since we're checking the git tag value, which contains "v".